lib/elixir/pages/references/unicode-syntax.md

```diff
@@ -136,12 +136,16 @@ Elixir will not warn on confusability for identifiers made up exclusively of cha
 
 ### C3. Mixed Script Detection
 
-Elixir will not allow tokenization of mixed-script identifiers unless the mixing is one of the exceptions defined in UTS 39 5.2, 'Highly Restrictive'. We use the means described in Section 5.1, Mixed-Script Detection, to determine if script mixing is occurring, with the modification documented in the section 'Additional Normalizations', below.
+Elixir will not allow tokenization of mixed-script identifiers unless it is via chunks separated by an underscore, like `http_сервер`, or unless the mixing within each of those chunks is one of the exceptions defined in UTS 39 5.2, 'Highly Restrictive'. We use the means described in Section 5.1, Mixed-Script Detection, to determine if script mixing is occurring, with the modification documented in the section 'Additional Normalizations', below.
```

Comment:
Now that we support chunks, I wonder if we should impose single scripts within the chunks? Highly restrictive is mostly about supporting `Latn + Jpan`, `Latn + Hanb`, `Latn + Kore`, but the Latin bit can be supported by underscore now.

Reply:
I have a dim memory that those writing systems actually incorporate latin characters into some of their words, with the example that comes to mind being the Japanese word for 'T-shirt'.

But it's been a while since I looked, and the actual extent of that valid usage is a question as well

Reply:
I have done a cursory look and it seems it mostly happens with prefixes, so it may be fine. But we can have this discussion in a separate pull request, once this is merged. It is not a lot of code to support highly restrictive either, but there may be a benefit in simplifying it conceptually.